membershipextras.php

```diff
@@ -247,6 +247,10 @@ function membershipextras_civicrm_post($op, $objectName, $objectId, &$objectRef)
   }
 
   if ($objectName == 'MembershipPayment') {
+    $isPaymentPlanPayment = _membershipextras_isPaymentPlanWithAtLeastOneInstallment();
```

Comment:
_membershipextras_isPaymentPlanWithAtLeastOneInstallment expect certain HTTP parameters to be passed and these won't exist if a webform is being submitted, so I am worried that this code will affect webforms, can you create a webform and see how this work with and without this.